cmd/builder/.goreleaser.yml

```diff
@@ -103,9 +103,6 @@ docker_manifests:
       - ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-builder:{{ .Env.CONTAINER_IMAGE_EPHEMERAL_TAG }}-ppc64le
 release:
   make_latest: false
-  github:
-    owner: open-telemetry
-    name: opentelemetry-collector-releases
```

Comment:
Why is this removed?

Reply:
I would rather do this change in a separate PR :sweat_smile: I think this is something we can merge before the next release, and then the nightlies PR can be merged right after the next release

Reply:
sure, i'll move that change out

Reply:
https://github.com/open-telemetry/opentelemetry-collector-releases/pull/1041